graduated/helm/step.yaml

```
@@ -203,16 +203,22 @@ spec:
                 "type": "boolean",
                 "description": "Use debian based cfstep-helm image",
                 "default": false
+            },
+            "image": {
+                "type": "string",
+                "description": "Override the default image used for the step"
             }
         }
     }
   stepsTemplate: |-
     main:
       name: helm
-      [[ if .Arguments.use_debian_image ]]
-      image: quay.io/codefreshplugins/cfstep-helm-debian:[[ .Arguments.helm_version ]]
+      [[ if .Arguments.image ]]
+        image: [[ .Arguments.image ]]:[[ .Arguments.helm_version ]]
+      [[ else if .Arguments.use_debian_image ]]
+        image: quay.io/codefreshplugins/cfstep-helm-debian:[[ .Arguments.helm_version ]]
```

```suggestion
      [[ if .Arguments.image ]]
      image: [[ .Arguments.image ]]:[[ .Arguments.helm_version ]]
      [[ else if .Arguments.use_debian_image ]]
      image: quay.io/codefreshplugins/cfstep-helm-debian:[[ .Arguments.helm_version ]]
```
Extra indentation breaks the step yaml. Image should be on the same level as the environment.